.github/workflows/e2e.yml

```
@@ -0,0 +1,84 @@
+name: e2e suite
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 2:00 AM UTC on Monday, Wednesday, and Friday
+    - cron: "00 2 * * 1,3,5"
+  pull_request:
+    branches:
+      - 'main'
+
+env:
+  BENCH_PATH: ./queries-results.json
+  DATABASE_URL: "postgres://saleor:saleor@postgres:5432/saleor"
+  SECRET_KEY: ci-test
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Freeze the version to avoid sudden unintended breaking changes:
```suggestion
    runs-on: ubuntu-22.04
```